po/genpot.sh

```diff
@@ -1,11 +1,4 @@
 #!/bin/sh
-# Stupid workaround for intltools not handling extensionless files
-ln -s terminator ../terminator.py
-ln -s remotinator ../remotinator.py
-
-# Make translation files
-intltool-update -g terminator -o terminator.pot -p
-
-# Cleanup after stupid workaround
-rm ../terminator.py
-rm ../remotinator.py
+cd ..
+python setup.py extract_messages
```

Comment:
Please drop this file, we should add the setup.py call to developer docs, e.g. `DEVELOPMENT.md`

Reply:
Sure.  Maybe a TRANSLATIONS.md file?  Does anyone have any best practices here?  Maybe I'll tag it on to the bottom of INSTALL.md for now.